### PR TITLE
export clusters number to prometheus

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
rebased @lichuqiang can you please readd a label ?

File: pilot/pkg/config/clusterregistry/secretcontroller.go [hunk 1]
```diff
@@ -41,6 +42,23 @@ const (
 	maxRetries = 5
 )
 
+var (
+	clusterCreationCounts = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "k8s_cluster_created_count",
```
Comment (douglas-reid) [inline pilot/pkg/config/clusterregistry/secretcontroller.go]: these should be prefixed by `pilot_` to match the other metrics from pilot.

Comment (douglas-reid) [reply pilot/pkg/config/clusterregistry/secretcontroller.go]: Also, please use prometheus naming conventions: https://prometheus.io/docs/practices/naming/#metric-names.

These should be named something like: `pilot_k8s_clusters_created_total` and `pilot_k8s_clusters_deleted_total`.

Comment (douglas-reid) [reply pilot/pkg/config/clusterregistry/secretcontroller.go]: Or, maybe `pilot_k8s_clusters_total` with label `action`: `{created | deleted}`

Comment (hzxuzhonghu) [reply pilot/pkg/config/clusterregistry/secretcontroller.go]: Thanks, addressed

**Comment by john-a-joyce** [comment]
We are removing the existing secret controller code so it can me a common library for Mixer and pilot.  This change should be applied to that common library not pilot library only.
/hold

**Pull request closed without merging**